### PR TITLE
PM-23 - Allow for multiple in-/outputs

### PR DESCRIPTION
**Pull request by Tabuu**
Stream Analytics is very powerful when having multiple in-/outputs. The variable naming also makes it easier to add other types of inputs and outputs later on.

This is also needed for PM-23.

Unfortunately we lose some default variables due to nesting, I commented the default values after the variable declaration. It will be possible to add default values for nested variables once the, currently experimental, feature is released.

### CODE REVIEW

File: modules/stream_analytics/main.tf [hunk 1]
```diff
@@ -13,20 +13,18 @@ provider "azurerm" {
 }
 
 locals {
-  stream_output_name = "${var.name}-output"
-  stream_input_name  = "${var.name}-input"
-  stream_job_name    = "${var.name}-job"
-
-  select_all_query = "SELECT * INTO \"${local.stream_output_name}\" FROM \"${local.stream_input_name}\""
```
Comment (godzillar) [inline modules/stream_analytics/main.tf:20]: Maybe we still need to have the default query here?

**Review by godzillar — COMMENTED** [review]
just one question.

File: modules/stream_analytics/main.tf (hunk 1 above)
Comment (Tabuu) [reply modules/stream_analytics/main.tf:20]: The default query can not longer be used since there can now be multiple in- and outputs

Comment (jvanenckevort) [reply modules/stream_analytics/main.tf:20]: I understand, any other options for a default query? Otherwise, leave it empty.

**Review by jvanenckevort — APPROVED** [review]

**Pull request merged**